components/FoodCard.js

```diff
@@ -6,15 +6,26 @@ export default function FoodCard({ recipe }) {
   return (
     <Card>
       <RecipeName>{recipe.name}</RecipeName>
-      <Image src={recipe.image} height={300} width={200} alt={recipe.name} />
+      <StyledImage src={recipe.image} height={300} width={200} alt={recipe.name} />
```

Comment:
I recommend to use the same size here as in your `ChooseFoodCard`, otherwise your images look kinda stretched.